triton/client/decode_manifest_triton.py

```diff
@@ -0,0 +1,416 @@
+#!/usr/bin/env python3
+# Copyright      2022  Xiaomi Corp.        (authors: Fangjun Kuang)
+#                2022  Nvidia              (authors: Yuekai Zhang)
+# See LICENSE for clarification regarding multiple authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This script loads a manifest in lhotse format and sends it to the server
+for decoding, in parallel.
+
+Usage:
+# For offline icefall server (default lang is Chinese)
+./decode_manifest_triton.py \
+    --compute-cer
+
+# For streaming icefall server
+./decode_manifest_triton.py \
+    --streaming \
+    --compute-cer
+
+# For offline wenet server
+./decode_manifest_triton.py \
+    --server-addr localhost \ 
+    --compute-cer \
+    --model-name attention_rescoring \
+    --num-tasks 300 \
+    --manifest-filename /mnt/samsung-t7/yuekai/aishell-test-dev-manifests/data/fbank/aishell_cuts_test.jsonl.gz
+
+# For streaming wenet server
+./decode_manifest_triton.py \
+    --server-addr localhost \ 
+    --streaming \
+    --compute-cer \
+    --context 7 \
+    --model-name streaming_wenet \
+    --num-tasks 300 \
+    --manifest-filename /mnt/samsung-t7/yuekai/aishell-test-dev-manifests/data/fbank/aishell_cuts_test.jsonl.gz
+
+
+
+(Note: You have to first start the server before starting the client)
+"""
+
+import argparse
```

Comment:
I suggest that you use `isort` to sort the import to follow the `pep8` style guide.